From 968c85eac21d4ea490d4fc095d6ce9b58381848a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Tue, 23 Aug 2022 17:03:47 +0530
Subject: [PATCH] Fix include search paths

I am not a cmake expert but setting the public interface causes
conda-forge not to find things such as <vector> anymore.
---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 73b7e38..f78e842 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -104,7 +104,7 @@ set(libclang_source
 
 add_library(cppast ${detail_header} ${header} ${source} ${libclang_source})
 target_compile_features(cppast PUBLIC cxx_std_11)
-target_include_directories(cppast PRIVATE ../include SYSTEM INTERFACE ../include)
+target_include_directories(cppast PUBLIC ../include)
 target_link_libraries(cppast PUBLIC type_safe tiny-process-library ${llvm_libs} clang)
 
 target_compile_definitions(cppast PUBLIC
-- 
2.37.1

